src/index.html

```diff
@@ -20,8 +20,434 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <header class="header">
+        <div>
+          <a
+            href="#"
+            class="img"
+          >
+            <img
+              src="images/logo.png"
+              alt="logo"
+            />
+          </a>
+        </div>
+        <nav class="nav">
+          <a
+            class="nav__link is-active"
+            href="#"
+          >
+            Apple
+          </a>
+          <a
+            class="nav__link"
+            href="#"
+          >
+            Samsung
+          </a>
+          <a
+            class="nav__link"
+            href="#"
+          >
+            Smartphones
+          </a>
+          <a
+            class="nav__link nav__link--laptops"
+            data-qa="nav-hover"
+            href="#"
+          >
+            Laptops & Computers
+          </a>
+          <a
+            class="nav__link"
+            href="#"
+          >
+            Gadgets
+          </a>
+          <a
+            class="nav__link"
+            href="#"
+          >
+            Tablets
+          </a>
+          <a
+            class="nav__link"
+            href="#"
+          >
+            Photo
+          </a>
+          <a
+            class="nav__link"
+            href="#"
```

Comment:
The `href` attributes for the navigation links are set to `#`, which means they won't navigate to any specific page. Ensure these links are correctly set to the intended destinations.